doc/man3/DES_random_key.pod

```diff
@@ -282,6 +284,17 @@ functions directly.
 Single-key DES is insecure due to its short key size.  ECB mode is
 not suitable for most applications; see L<des_modes(7)>.
 
+=head1 RETURN VALUES
+
+DES_set_key(), DES_key_sched(), DES_set_key_checked() and DES_is_weak_key()
+return 0 on success or negative values on error.
+
+DES_cbc_cksum() and DES_quad_cksum() return 4-byte integer representing the
+last 4 bytes of the checksum of the input.
+
+DES_fcrypt() returns a pointer to the caller-provided buffer and DES_crypt() -
+to a static buffer on success; otherwise they return NULL.
```

Comment:
It looks like I was concentrating on wrong "a", huh? I mean initially I was tipped off by "a string", but it's still "*a* pointer", but to "*the* buffer". Sorry about possible confusion...

Reply:
I don't quite get the point, could you explain a bit more? Seems it's fine saying 'a point to the buffer'....

Reply:
Point is that my English sucks. I mean originally I made fuzz about "a" being indefinite article as if it has *no* bearing here. But it apparently does, "*a* pointer to *the* buffer"... Oh well... It was inessential comment...

Reply:
Ahh...